### DISCUSSION TITLE
blog/web-tasks

### DISCUSSION THREAD

**Discussion by giscus** [GawdOfGithub/dcc-blog#5] [General]
# blog/web-tasks

https://blog.dccnita.in/blog/web-tasks

**Comment by ghost** [comment]
Can we use other APIs rather than Memegen ?

**Reply by Edantuti** [reply]
Yes you can. We provided memgen api because it had edit functionality.

**Comment by ghost** [comment]
Can I use pexels API for task 2 since the verification for Unsplash API will take 2-10 business days?

**Reply by Edantuti** [reply]
You can
